Source/main.cpp

```diff
@@ -700,7 +698,7 @@ void main_main ()
 	// Copy the intermediate values to the next sub-iteration
         for ( int comp=0; comp < AMREX_SPACEDIM; ++comp)
         {
-            MultiFab::Copy(velImPrev[comp], velCont[comp], 0, 0, 1, 0);
+            MultiFab::Copy(  velImRK[comp],     velCont[comp], 0, 0, 1, 0);
```

Comment:
I simplified the code by copying the initial condition into velImRK here, and the at the beginning of each iteration copying velImRK into velIMPrev (and skipping the copy at the end).  I verified the solution evolves identically.